### PR TITLE
Fix project order for 1.9

### PR DESCRIPTION
**Pull request by gdalle**
Solves #105

### CODE REVIEW

**Comment by Krastanov** [comment]
@oxinabox , @DilumAluthge -- pardon the cold ping, but you are the most senior active folks in this organization. Any chance you can merge this and make a new release? Aqua checks are failing on pkgextensions

**Comment by gdalle** [comment]
Not sure how to fix code style though, sorry

File: src/utils.jl [hunk 1]
```diff
@@ -97,7 +97,19 @@ catch
...
+    "uuid",
+    "keywords",
+    "license",
+    "desc",
+    "deps",
+    "weakdeps",
+    "extensions",
+    "compat",
+    "extras",
+    "targets"
```
Comment (DilumAluthge) [inline src/utils.jl:111]:
```suggestion
    "targets",
```

**Review by DilumAluthge — APPROVED** [review]

**Comment by gdalle** [comment]
thanks!

**Comment by jishnub** [comment]
This should perhaps be backported to v0.5

**Pull request merged**